public/app/plugins/datasource/mssql/partials/config.html

```diff
@@ -1,80 +1,83 @@
-
 <h3 class="page-heading">MS SQL connection</h3>
 
 <div class="gf-form-group">
-	<div class="gf-form max-width-30">
-		<span class="gf-form-label width-7">Host</span>
-		<input type="text" class="gf-form-input" style="width: 352px" ng-model='ctrl.current.url' placeholder="localhost" bs-typeahead="{{['localhost', 'localhost" required></input>
```

Comment:
this line is the only one that changed, the bs-typeahead line was an incomplete expression. Then prettier formatted the whole document :)